output.json

```diff
@@ -1,36 +1,40 @@
 {
-    "blueprints": [
-        {
-            "invoke": "colors.pr",
-            "write_to": "colors.ts"
-        },
-        {
-            "invoke": "borders.pr",
-            "write_to": "borders.ts"
-        },
-        {
-            "invoke": "gradients.pr",
-            "write_to": "gradients.ts"
-        },
-        {
-            "invoke": "measures.pr",
-            "write_to": "measures.ts"
-        },
-        {
-            "invoke": "shadows.pr",
-            "write_to": "shadows.ts"
-        },
-        {
-            "invoke": "radii.pr",
-            "write_to": "radii.ts"
-        },
-        {
-            "invoke": "typography.pr",
-            "write_to": "typography.ts"
-        },
-        {
-            "invoke": "theme.pr",
-            "write_to": "theme.ts"
-        }
-    ]
-}
\ No newline at end of file
+  "blueprints": [
+    {
+      "invoke": "colors.pr",
+      "write_to": "colors.ts"
+    },
+    {
+      "invoke": "buttonColors.pr",
```

Comment:
As discussed over Slack - It might be better to just include all the option tokens that a specific component, like this Button, will need in order for it to construct itself. That way, we don't have multiple files for different option tokens for a single component (i.e. separate files for color, shadows, gradients, dimensions, etc).